catchup/service.go

```diff
@@ -147,6 +156,32 @@ func (s *Service) IsSynchronizing() (synchronizing bool, initialSync bool) {
 	return
 }
 
+// SetSyncRound attempts to set the minimum sync round to keep in the cache
+func (s *Service) SetSyncRound(rnd uint64) error {
+	if basics.Round(rnd) < s.ledger.LastRound() {
+		return ErrSyncRoundInvalid
+	}
+	s.syncRoundMu.Lock()
+	defer s.syncRoundMu.Unlock()
+	s.syncRound = rnd
+	return nil
+}
+
+// UnsetSyncRound removes any previously set sync round
+func (s *Service) UnsetSyncRound() error {
```

Comment:
@Eric-Warehime Minor:  `UnsetSyncRound` and `GetSyncRound` do not produce errors.  Is the `error` return type retained by-design or can it be removed?

Reply:
@Eric-Warehime Resolving - Thanks for bringing in the feedback via https://github.com/algorand/go-algorand/pull/4658/commits/4114c3b5efc51a83181dc4693ba91f63d0033934.